samza-kv/src/main/scala/org/apache/samza/storage/kv/BaseKeyValueStorageEngineFactory.scala

```diff
@@ -35,15 +35,22 @@ import org.apache.samza.util.ScalaJavaUtil.JavaOptionals
 import org.apache.samza.util.{HighResolutionClock, ScalaJavaUtil}
 
 /**
- * A key value storage engine factory implementation
- *
- * This trait encapsulates all the steps needed to create a key value storage engine. It is meant to be extended
- * by the specific key value store factory implementations which will in turn override the getKVStore method.
- */
+  * A key value storage engine factory implementation
+  *
+  * This trait encapsulates all the steps needed to create a key value storage engine. It is meant to be extended
+  * by the specific key value store factory implementations which will in turn override the getKVStore method.
+  */
 trait BaseKeyValueStorageEngineFactory[K, V] extends StorageEngineFactory[K, V] {
 
   private val INMEMORY_KV_STORAGE_ENGINE_FACTORY =
     "org.apache.samza.storage.kv.inmemory.InMemoryKeyValueStorageEngineFactory"
+  private val CHANGELOG_MAX_MSG_SIZE_BYTES = "changelog.max.message.size.bytes"
```

Comment:
Consider moving these configs to `StorageConfig`.
Also, it would probably be good to be able to configure these per store (see `StorageConfig`).

Reply:
Done.